### PR TITLE
chore: updated shared lib

### PR DESCRIPTION
**Pull request by vijayg10**
feat(mojaloop/[#2092](https://github.com/mojaloop/project/issues/2092)): upgrade nodeJS version for core services - https://github.com/mojaloop/project/issues/2092
- Upgrade to node v16
- Update packages/linting/audit/dockerfile
- Update ci with dep/audit checks

BREAKING CHANGE: Major version bump for node v16 LTS support.

Major version bump since this is a big upgrade.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -85,16 +82,17 @@
...
-    "eslint": "^8.17.0",
+    "audit-ci": "^6.3.0",
+    "eslint": "^8.20.0",
     "eslint-plugin-import": "^2.26.0",
-    "jest": "^28.1.0",
-    "jest-junit": "^13.2.0",
-    "jsdoc": "^3.6.10",
+    "jest": "^28.1.3",
+    "jest-junit": "^14.0.0",
     "npm-audit-resolver": "3.0.0-7",
```
Comment (mdebarros) [inline package.json:90]: Please remove, and re-run `npm install`

```suggestion
```

File: package.json [hunk 2]
```diff
@@ -85,16 +82,17 @@
     "ws": "^7.5.5"
   },
   "devDependencies": {
-    "eslint": "^8.17.0",
+    "audit-ci": "^6.3.0",
+    "eslint": "^8.20.0",
```
Comment (mdebarros) [inline package.json:86]: Why do we have `eslint` here if we are using `standard`?

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@mojaloop/ml-testing-toolkit-client-lib",
   "description": "Testing Toolkit Client Library",
-  "version": "0.0.6",
+  "version": "1.0.0-snapshot.0",
```
Comment (mdebarros) [inline package.json:4]: You are missing `engines` in this package.json (e.g. https://github.com/mojaloop/central-ledger/blob/master/package.json#L26)

```
  "engines": {
    "node": "=16.x"
  },
```

**Review by mdebarros — CHANGES_REQUESTED** [review]
See some minor suggestions.

File: package.json (hunk 1 above)
Comment (vijayg10) [reply package.json:90]: Done

File: package.json (hunk 3 above)
Comment (vijayg10) [reply package.json:4]: Done

File: package.json (hunk 2 above)
Comment (vijayg10) [reply package.json:86]: Done

**Pull request merged**